vignettes/articles/prepare-loans-inputs.Rmd

```diff
@@ -71,8 +74,9 @@ achieve this, instead of the previous command, simply run:
 
 ```{r}
 run_prep_calculation_loans(
-  data_path_project_specific = "/path/to/specific/data",
-  data_path_project_agnostic = "/path/to/agnostic/data",
+  input_path_project_specific = "/path/to/specific/data",
+  input_path_project_agnostic = "/path/to/agnostic/data",
+  data_prep_output_path = "/path/to/specific/data",
   credit_type = "credit_limit"
 )
```

Comment:
Run? Prepare? or Calculate? What's the action here?

I suggest `prep_loans()`.